Library/Homebrew/cmd/test-bot.rb

```diff
@@ -480,6 +480,7 @@ def formula formula_name
       test "brew", "install", "--build-bottle", *changed_dependences unless changed_dependences.empty?
       formula_fetch_options = []
       formula_fetch_options << "--build-bottle" unless ARGV.include? "--no-bottle"
+      formula_fetch_options << "--build-from-source" unless ARGV.include? "--build-bottle"
```

Comment:
Yeah, I did, sorry. Was tired.

I think I'll just scrap it. If edge cases come up we can handle them once we actually know what those edge cases are, rather than trying to guess them in advance.